pkg/cluster/internal/meta/meta.go

```diff
@@ -0,0 +1,62 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package meta
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"sigs.k8s.io/kind/pkg/cluster/consts"
+)
+
+// ClusterMeta contains some cluster meta and can be used to compute some more
+// cluster metadata
+// See: NewClusterMeta
+type ClusterMeta struct {
+	name string
+}
+
+// NewClusterMeta returns a new cluster meta
+func NewClusterMeta(name string) *ClusterMeta {
+	return &ClusterMeta{
+		name: name,
+	}
+}
+
+// Name returns the cluster's name
+func (c *ClusterMeta) Name() string {
+	return c.name
+}
+
+// KubeConfigPath returns the path to where the Kubeconfig would be placed
+// by kind based on the configuration.
+func (c *ClusterMeta) KubeConfigPath() string {
+	// TODO(bentheelder): Windows?
```

Comment:
`%HOME%/.kube` is a valid path on win32.
this function is portable, so we can omit the TODO?

Reply:
ok, i guess that's true. it was set for me on Windows 7 (explicitly), but not on a Windows XP VM and also on Windows 10.

Reply:
this matches upstream now: https://github.com/kubernetes/client-go/blob/ee7a1ba5cdf1292b67a1fdf1fa28f90d2a7b0084/tools/clientcmd/loader.go#L51